### PR TITLE
feat: Add road to performance blog post

### PR DESCRIPTION
**Pull request by wouterlucas**

### CODE REVIEW

**Review by erikhaandrikman — APPROVED** [review]

File: site/blogs/roadToPerformance.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+author: Wouter-lucas van Boesschoten
+---
+
+# Blits: Unlocking Performance in Your Big Screen Experience
+
+**Performance-obsessed technology for contained rendering.**
+
+Blits is a brand new (well, 1 year old at the time of writing) JavaScript Application Development UI library for Lightning 3. Where Lightning 2 had a built-in application development syntax, Lightning 3 is split into two main building blocks: the renderer and Blits. This separation provides more flexibility and iterability between the two foundational pillars behind Lightning 3.
+
+The Smart TV, game console, and set-top box space is a niche within a niche, and Lightning is the key player in that space. Most platforms come with a form of browser—usually far older than the one you’re using to read this. The one feature that ties them together is support for WebGL, which makes rendering stable across a wide array of devices, models, and years—dating from TVs running browsers built in 2014 to more recent versions.
```
Comment (jfboeve) [inline site/blogs/roadToPerformance.md:15]: browser-usually and years-dating seem a bit off

File: site/blogs/roadToPerformance.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+But why is this a niche? Well, the performance of a Smart TV is somewhere between 10x to 100x slower than your average mobile device. This means that any performance gain, whether small or big, makes a major difference in running your TV experience on such a slow device. With that in mind, we created Lightning 3's Blits. A pure obsession with performance has resulted in an unparalleled application development framework with best-in-class efficiency.
+
+# The Lost Art of Performance
+
+Although the trend of "going green" is slowly bringing the art of writing more performant code back into the spotlight, it's important to note the shift. The post-2000s and 2010s saw wild growth in endless resources, but these are now increasingly becoming a concern due to their carbon footprint. Efforts to reduce this are pushing tech to be more and more "green." However, in our niche market of highly resource-constrained TV devices, the challenges are on a whole different level. This isn't just about going green—it's like we're back in the 80s, where you're working with a very constrained device and need to write a version of Mario Kart that runs consistently above 24 FPS.
```
Comment (jfboeve) [inline site/blogs/roadToPerformance.md:21]: green-it's

File: site/blogs/roadToPerformance.md (hunk 1 above)
Comment (wouterlucas) [reply site/blogs/roadToPerformance.md:15]: yeah it started with stuff between brackets, then between -'s to emphasize a small contextual change in the narrative but the sentence in between got too big :D 

thanks! will fix

File: site/blogs/roadToPerformance.md (hunk 2 above)
Comment (wouterlucas) [reply site/blogs/roadToPerformance.md:21]: thanks! good spot

File: site/blogs/roadToPerformance.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+The art of writing performant code for super-constrained devices slowly died in the late 90s. Before that, game systems like Atari, Nintendo, and early PC game engines were masterpieces of performance optimization, pushing the boundaries of what was possible. These efforts often led to crazy code patterns, many at the expense of readability (sorry, TypeScript). But with the rise of systems like the PlayStation 3, Xbox 360, and more powerful gaming PCs (RIP Voodoo Banshee with 3dfx Glide), the need for such intricate optimization began to fade.
+
+We've had a few sessions where we revisited old 80s/90s game code just for the learnings. Sure, applying the same patterns in JavaScript isn’t always compatible—the language is just higher level than old C code—but some patterns still apply, and valuable lessons can be learned. By gaining a deeper understanding of the JIT tiers and how JavaScript engines interpret code, we were able to push performance for Lightnign 3 Blits to its limits. This was no easy feat, and there’s always more to optimize, but it did lead to the fastest contained rendering system available for JavaScript.
```
Comment (michielvandergeest) [inline site/blogs/roadToPerformance.md:25]: 2 typo's:

"compatible - the"
Lightning

Comment (wouterlucas) [reply site/blogs/roadToPerformance.md:25]: "compatible - the" is part of a context switch:
compatible—the language is just higher level than old C code—but some patterns still apply

so the —the language is just higher level than old C code— is like doing 
(the language is just higher level than old C code)

but nicer? 
I used to do ()'s all the time but trying to use -'s more often... 🤷 

and LOL on Lightnign 😎

**Pull request merged**